### PR TITLE
Default arguments for post_alert

### PR DESCRIPTION
**Pull request by mcoughlin**
This PR adds default arguments for post_alert.

### CODE REVIEW

File: extensions/skyportal/skyportal/handlers/api/alert.py [hunk 1]
```diff
@@ -245,18 +257,26 @@ def post_alert(
...
         candids = {a.get("candid", None) for a in alert_data["prv_candidates"]}
         if latest_alert_data["candidate"]["candid"] not in candids:
             alert_data["prv_candidates"].append(latest_alert_data["candidate"])
 
-    df = pd.DataFrame.from_records(alert_data["prv_candidates"])
-    mask_candid = df["candid"] == str(candid)
-
-    if candid is None or sum(mask_candid) == 0:
+    if candid is None:
         candid = int(latest_alert_data["candidate"]["candid"])
```
Comment (guynir42) [inline extensions/skyportal/skyportal/handlers/api/alert.py:268]: what if `latest_alert_data` is an empty dict here?

File: extensions/skyportal/skyportal/handlers/api/alert.py [hunk 2]
```diff
@@ -567,7 +589,7 @@ async def get(self, object_id: str = None):
                           data:
                             type: array
                             items:
-                              type: object
+                               type: object
```
Comment (guynir42) [inline extensions/skyportal/skyportal/handlers/api/alert.py:592]: This looks like an accidental space (there should be two spaces)

File: extensions/skyportal/skyportal/handlers/api/alert.py [hunk 3]
```diff
@@ -647,6 +669,7 @@ async def get(self, object_id: str = None):
 
             if response.get("status", "error") == "success":
                 alert_data = response.get("data")
+                print(alert_data)
```
Comment (guynir42) [inline extensions/skyportal/skyportal/handlers/api/alert.py:672]:
```suggestion
```

**Review by guynir42 — APPROVED** [review]
Some minor comments. Thanks for fixing this up!

File: extensions/skyportal/skyportal/handlers/api/alert.py (hunk 2 above)
Comment (mcoughlin) [reply extensions/skyportal/skyportal/handlers/api/alert.py]: Good call.

File: extensions/skyportal/skyportal/handlers/api/alert.py (hunk 3 above)
Comment (mcoughlin) [reply extensions/skyportal/skyportal/handlers/api/alert.py]: Removed.

File: extensions/skyportal/skyportal/handlers/api/alert.py (hunk 1 above)
Comment (mcoughlin) [reply extensions/skyportal/skyportal/handlers/api/alert.py:270]: Good idea, adding an extra check there.

**Pull request merged**